### DISCUSSION TITLE
Disabling command pallette results in taken shift and control key over all platform regardless of text input

### DISCUSSION THREAD

**Discussion by hacknorris-code** [community/community#24056] [Other Feature Feedback, Questions, & Ideas]
<p>so yes, just disabled command pallette to use my fav vim keybinds from browser (was messing up with this and i luv vim). so i ended up that when creating issue, searching github or doing an other thing when i press shift key (to for example type colon or mention someone)  i get command palette… possible to fix? are there rules for adguard? (block command pallette thing from source)</p>

**Comment by dschwilk** [comment, marked as answer]
<p>There is one other place to turn off the command palette completely: Under “feature preview” in the drop down menu obtained from clicking on your user icon upper right. I finally found this and saw that I could opt out of this “beta” feature. Turning command palette off in the “settings” did NOT work.  So this is a solution for me – at least until github decides this is a mandatory feature and not “beta”.  Github was completely unusable for me until I discovered this.</p>

**Comment by hacknorris-code** [comment]
<p>already figured it out. yea, only this works… <span class="spoiler">if github will make it a default feature i think i’ll need to fully switch to codeberg… not want, not mirror repos… fully switch…</span></p>
